### PR TITLE
REF: Update balance view to new UX

### PR DESCRIPTION
**Pull request by marcosrdz**
Screenshot 2024-03-31 at 12 33 41 PM

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/36ma6byoe53neifdba7zvl4e6a?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1027)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/s7uayukqrtxjgbupls3kwe7fu4?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1028)

File: screen/wallets/transactions.js [hunk 1]
```diff
@@ -510,6 +512,20 @@ const WalletTransactions = ({ navigation }) => {
             saveToDisk();
           })
         }
+        onWalletBalanceVisibilityChange={async isShouldBeVisible => {
+          const isBiometricsEnabled = await Biometric.isBiometricUseCapableAndEnabled();
+          if (wallet.hideBalance) {
+            if (isBiometricsEnabled) {
+              if (!(await Biometric.unlockWithBiometrics())) {
```
Comment (Overtorment) [inline screen/wallets/transactions.js:519]: 3 consecutive IFs, can this be simplified?

**Review by Overtorment — APPROVED** [review]
LGTM

File: screen/wallets/transactions.js (hunk 1 above)
Comment (marcosrdz) [reply screen/wallets/transactions.js]: Done

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/tca26j2egdyymgybddepc2z4mu?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1031)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/dh7pldwix6wuw27s5fuaslbov4?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1037)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/ktortuhsqnfkmdryl33nc5h2ty?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1040)

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**